src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/AssignCaseAccessService.java

```diff
@@ -33,4 +39,11 @@ public void assignCaseAccess(CaseDetails caseDetails, String authorisationToken)
             assignCaseAccessRequest
         );
     }
+
+    public AboutToStartOrSubmitCallbackResponse applyDecision(String authToken, CaseDetails caseDetails) {
+        log.info("Updating case access via assignCaseAccessService for caseID {}", caseDetails.getId());
+        log.info("Sending payload to ManageCaseAssignment Service: {}", caseDetails);
```

Comment:
why 2 method calls for logging? could just add a \n then the next message

Reply:
These are to be deleted anyway, this wasn't quite ready for review - print statements and logging of sensitive data is all to go before merge